src/MobileUI/AppShell.xaml

```diff
@@ -179,8 +179,8 @@
         </Tab>-->
 
 
-        <Tab Route="people" Icon="icon_people" Title="SSW People">
-            <ShellContent ContentTemplate="{DataTemplate pages:PeoplePage}" />
+        <Tab Route="people" Icon="icon_people" Title="Networking">
```

Comment:
What is in the design for this, is it "Network" or "Networking"?

Reply:
It is "Network" I've updated everything appropriately :)